src/wrapper/build.rs

```diff
@@ -292,6 +301,9 @@ pub fn build_container_cmd(wrapper: &Wrapper, cmdline: Vec<String>)
         ap.refer(&mut force)
             .add_option(&["--force"], StoreTrue,
                 "Force build even if container is considered up to date");
+        ap.refer(&mut no_image)
+            .add_option(&["--no-image"], StoreTrue,
```

Comment:
Similarly to the above. In this case simpler `--no-download` may be good enough.